simple_learn/classifiers/simple_classifier.py

```diff
@@ -30,6 +30,7 @@
 from sklearn.metrics import f1_score, jaccard_score
 from sklearn.model_selection import GridSearchCV
 from sklearn.utils import all_estimators
+from tqdm import tqdm
```

Comment:
Is `tqdm` part of the python standard library? If not can you add it to requirements.txt

Reply:
Nope it's a library so I'll do that quick.